rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -152,6 +155,93 @@ void voteFeePerKbChange_success() {
         assertEquals(FeePerKbResponseCode.SUCCESSFUL_VOTE.getCode(), result);
     }
 
+    @Test
+    void getLockWhitelistSize() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        when(whitelistSupport.getLockWhitelistSize()).thenReturn(Integer.SIZE);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withWhitelistSupport(whitelistSupport)
+            .build();
+
+        assertEquals(Integer.SIZE, bridgeSupport.getLockWhitelistSize());
+    }
+
+    @Test
+    void getLockWhitelistEntryByIndex() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        LockWhitelistEntry entry = mock(LockWhitelistEntry.class);
+        when(whitelistSupport.getLockWhitelistEntryByIndex(Integer.SIZE)).thenReturn(entry);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withWhitelistSupport(whitelistSupport)
+            .build();
+
+        assertEquals(entry, bridgeSupport.getLockWhitelistEntryByIndex(Integer.SIZE));
+    }
+
+    @Test
+    void getLockWhitelistEntryByAddress() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        LockWhitelistEntry entry = mock(LockWhitelistEntry.class);
+        when(whitelistSupport.getLockWhitelistEntryByAddress(any())).thenReturn(entry);
```

Comment:
nit: can replace `any()` by `"address"` to make the test more explicit. Same for the other tests.